examples/types-use-ipfs-from-typed-js/src/main.js

```diff
@@ -17,7 +17,6 @@ async function main () {
 
   console.log('Added file:', file.path, file.cid.toString())
   try {
-    // @ts-expect-error CID has no toUpperCase method
```

Comment:
I think this defeats the point of the point of this example to a degree. It was there to ensure that TS is able to pick up types properly. Unfortunately it got broken, but we should fix not remove the check.

I think it would be better to temporarily disable this example test instead of just removing this line.

Reply:
The `@ts-expect-error` was only added [yesterday](https://github.com/ipfs/js-ipfs/pull/3447) so the purpose of this this example has only ever been defeated.

The it's never run in CI because the module name was not picked up by the [scope filter](https://github.com/ipfs/js-ipfs/blob/master/.travis.yml#L243).

Please can you submit a fix in a follow up PR.  This one cannot wait any longer.